src/components/common/form/InputFile.web.js

```diff
@@ -20,69 +18,31 @@ const toBase64 = file =>
     reader.onerror = error => reject(error)
   })
 
+const getImageWithSrc = src => {
+  return new Promise((resolve, reject) => {
```

Comment:
how about prefetch ? have you tested the following code (i've sent you in the Slack) ?

```javascript
import { Image as NativeImage } from 'react-native'

const image = new Image()

await NativeImage.prefetch(imageUrl)
image.src = imageUrl // should immediately load from cache

const resized = await callYourResizeFunction(image) 
```

if won't work please try also to move image.src = xxx before await